doc/nrf/ug_nrf5340.rst

```diff
@@ -139,7 +139,19 @@ Depending on the sample, you must program only the application core (for example
    You can use the :ref:`nrf5340_empty_app_core` sample for this purpose.
    For details, see the code in :file:`zephyr/boards/arm/nrf5340pdk_nrf5340/nrf5340_cpunet_reset.c`.
 
-Build and program both samples separately by following the instructions in :ref:`gs_programming_ses`.
+To program only the application core, follow the instructions in :ref:`gs_programming_ses` and use ``nrf5340pdk_nrf5340_cpuapp`` or ``nrf5340pdk_nrf5340_cpuappns`` as build target.
+
+When programming both the application core and the network core, you can choose whether you want to build and program both images separately or combined as a :ref:`multi-image build <ug_multi_image>`.
+
+In a multi-image build, the image for the application core is the parent image, and the image for the network core is treated as a child image in a separate domain.
+For this to work, the network core image must be explicitly added as a child image to one of the application core images.
+See :ref:`ug_multi_image_defining` for details.
+
+The network sample :ref:`zephyr:bluetooth-hci-rpmsg-sample` is automatically added to all Bluetooth Low Energy samples in the |NCS|.
+When :option:`CONFIG_BT_RPMSG_NRF53` is set to ``y`` (the default), the build system automatically includes the sample as a child image in the ``nrf5340_pdk_nrf5340_cpunet`` core.
+You can turn this off by setting :option:`CONFIG_BT_RPMSG_NRF53` to ``n``.
```

Comment:
Does this work? The description of the option doesn't say anything about multi-image builds, so what else would you be turning off in this case?

Reply:
Yes this works. 

The option is upstream and our build scheme for multi image is only downstream. 

I guess we could append some help text by having a duplicate entry downstream if you think it is needed.

Reply:
That would be nice, but my main concern is if turning this off will cause anything else except for the multi-image build to be turned off. Because the option probably does something upstream as well?

Reply:
Good point. For now you can not disable this feature without modifying the CMake, which I guess is something we don't document. Support for kconfig this out will be added after the release.

Reply:
I removed this line.

Reply:
Wait, so what happens if you set the kconfig option to "n"?!

Reply:
Then you will no longer be able to use rpmsg as the hci backend, and we will not include hci_rpmsg sample as a child image.